cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -713,12 +716,21 @@ def _configure_cluster(self):
                         members=valid_names,
                     )
                 )
-        etcd_name_suffix = etcd_name_suffix.replace('.', '_')
+        ip_urlized = '[{0}]'.format(private_ip) if network.is_ipv6(private_ip)\
+            else socket.gethostbyname(private_ip)
+        cluster_nodes = {k: v for k, v in
+                         config[POSTGRESQL_SERVER]['cluster']['nodes'].items()}
+        for k, v in cluster_nodes.items():
+            if 'ip' in v:
+                v['ip'] = network.ipv6_url_compat(v['ip'])
```

Comment:
Why do you use this here, and not in the ip_urlized above?

Reply:
`ip_urlized` is used for current node (and sometimes uses `gethostbyname()` which is only falid for IPv4)
`ipv6_url_compat` works on already defined addresses of all the nodes